### PR TITLE
chore: Bump nav_version and set jvm target to 1.8

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #2141 

Reference issue: https://issuetracker.google.com/issues/137782949
Solution: https://issuetracker.google.com/issues/137268935

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -94,13 +94,17 @@ android {
     }
     lintOptions {
         disable 'MissingTranslation'
+        disable 'InvalidPackage'
```
Comment (ShridharGoel) [inline app/build.gradle]: Why?

Comment (ShridharGoel) [reply app/build.gradle]: If this cannot be avoided then consider using `warning 'InvalidPackage'`
https://stackoverflow.com/questions/21827004/gradle-dagger-lint-ignore-by-package#comment58243384_21985399

Comment (liveHarshit) [reply app/build.gradle]: Updated. Another solution is: https://github.com/getsentry/sentry-java/issues/571#issuecomment-378295015

**Comment by iamareebjamal** [comment]
Test app on API < 24

**Comment by liveHarshit** [comment]
> Test app on API < 24

Tested in API 23, working fine.

**Pull request merged**